testapp/src/main/res/layout/item_section_miniapp.xml

```diff
@@ -2,44 +2,36 @@
 <layout>
 
     <data>
-        <import type="android.text.TextUtils"/>
+
+        <import type="android.text.TextUtils" />
+
         <variable
             name="miniapp"
             type="com.rakuten.tech.mobile.miniapp.MiniAppInfo" />
     </data>
 
-    <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+    <androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
         xmlns:tools="http://schemas.android.com/tools"
-        xmlns:app="http://schemas.android.com/apk/res-auto">
-
-        <View
-            android:layout_width="match_parent"
-            android:layout_height="1dp"
-            android:background="@color/color_bg_default_icon"
-            app:layout_constraintTop_toTopOf="parent"/>
-
-        <View
-            android:layout_width="match_parent"
-            android:layout_height="1dp"
-            android:background="@color/color_bg_default_icon"
-            app:layout_constraintBottom_toBottomOf="parent"/>
+        android:layout_width="match_parent"
+        android:background="@color/bg_section"
+        android:layout_height="wrap_content">
 
         <TextView
             android:id="@+id/tv_app_name"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:ellipsize="end"
+            android:padding="@dimen/small_5"
             android:singleLine="true"
             android:text="@{miniapp.displayName}"
             android:textColor="@android:color/black"
-            android:textSize="@dimen/text_large_16"
-            android:padding="@dimen/medium_16"
```

Comment:
I think that this still needs `dimen/medium_16` for left and right padding. The text looks too close to the edge of the screen.

Reply:
Okay, that's manageable by just the paddingStart/End. Modified.

Reply:
I have currently reset it to 8dp as it was giving a distinct factor, 16 was making it quite in line. 
Put all the 4 screenshots for reference to give a comparative view.

![image](https://user-images.githubusercontent.com/56627842/87145779-d0d25a00-c2e4-11ea-85d0-270b87e6ff65.png)
![image](https://user-images.githubusercontent.com/56627842/87145787-d62fa480-c2e4-11ea-8020-64bbd7e5abce.png)
![image](https://user-images.githubusercontent.com/56627842/87145793-da5bc200-c2e4-11ea-9552-ed8cacbb322f.png)
![image](https://user-images.githubusercontent.com/56627842/87145795-dcbe1c00-c2e4-11ea-9376-618d721965b4.png)

Let me know @corycaywood @minh-rakuten

Reply:
4️⃣ is good for me.

Reply:
I agree that 4 looks the best. 3 looks mis-aligned.

Reply:
Lol. Okay. 4th it is then.